src/pages/Movie/Movie.jsx

```diff
@@ -1,25 +1,77 @@
 import { useParams } from "react-router-dom";
 import { useEffect, useState } from "react";
 import { apiInstance } from "../../api/apiInstance";
+import { NavLink } from "react-router-dom";
+
+import styles from "./Movie.module.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCircleLeft,
+  faCircleRight,
+} from "@fortawesome/free-regular-svg-icons";
 
 export const Movie = () => {
   const { id } = useParams();
 
-  const [movieInfo, setMovieInfo] = useState(null);
+  const [movie, setMovie] = useState(null);
 
   const getMovie = async () => {
     const data = await apiInstance.get("/movie/" + id);
-    setMovieInfo(data.data);
-    console.log(data);
+    setMovie(data.data);
   };
 
   useEffect(() => {
     getMovie();
   }, []);
 
+  if (!movie) return <p>Loading...</p>;
+
   return (
-    <div>
-      <p>movie id = {id}</p>
+    <div className={styles.сontainer}>
+      <img
+        className={styles.backgroundImg}
+        src={
+          process.env.REACT_APP_API_IMG_URL + "/original" + movie.backdrop_path
+        }
+        alt={movie.title}
+      />
+      <div className={styles.ButtonContainer}>
+        <NavLink className={styles.link}>
```

Comment:
I see u didnt add any functionality for the links. Add the link to the main page for the 1st button and link to the next movie for the 2nd button (in case if the next movie is already in the array of movies already, use id of the next movie after the current one. In case if the current movie is the last one, do a request for the next page of the movies list and use the first movie id as the next one). 

P.S.: the functionality for the 2nd button we will implement when we will have a redux store, not its kinda weird to implement cause of props drilling